client/signup/steps/business-information/index.jsx

```diff
@@ -0,0 +1,148 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import { isUserLoggedIn } from 'state/current-user/selectors';
+
+//Form components
+import Card from 'components/card';
+import Button from 'components/button';
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextarea from 'components/forms/form-textarea';
+import FormLabel from 'components/forms/form-label';
+import FormFieldset from 'components/forms/form-fieldset';
+import InfoPopover from 'components/info-popover';
+
+class BusinessInformation extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			businessInformation: get(
+				this.props,
+				'signupProgress[3].providedDependencies.businessInformation',
+				{}
+			),
+		};
+	}
+
+	handleSubmit = event => {
+		event.preventDefault();
+		const { goToNextStep, stepName, flowName, translate } = this.props;
+
+		//Create site
+		SignupActions.submitSignupStep(
+			{
+				processingMessage: translate( 'Collecting your information' ),
+				stepName: stepName,
+			},
+			[],
+			{
+				businessInformation: this.state.businessInformation,
+			}
+		);
+
+		goToNextStep( flowName );
+	};
+
+	renderContent() {
+		const { translate } = this.props;
+
+		return (
+			<div className="business-information__wrapper">
+				<div className="business-information__form-wrapper ">
+					<form onSubmit={ this.handleSubmit }>
+						<Card>
+							<h3>{ translate( 'Basic Information' ) }</h3>
+							{ translate( 'How do people find and contact your business?' ) }
+
+							<FormFieldset>
+								<FormLabel htmlFor="name">
+									{ translate( 'Business Name' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Business name' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).